src/github/pullRequestManager.ts

```diff
@@ -771,6 +771,101 @@ export class PullRequestManager implements vscode.Disposable {
 		};
 	}
 
+	async getIssues(options: IPullRequestsPagingOptions = { fetchNextPage: false }): Promise<PullRequestsResponseResult> {
+		if (!this._githubRepositories || !this._githubRepositories.length) {

```

Comment:
Is it possible to extract this into something like `fetchPagedData` which could take a flag to determine if pull requests or issues should be requested?